src/contracts/core/DelegationManager.sol

```diff
@@ -657,63 +737,63 @@ contract DelegationManager is Initializable, OwnableUpgradeable, Pausable, Deleg
         emit WithdrawalCompleted(withdrawalRoot);
     }
 
-    // @notice Increases `operator`s delegated shares in `strategy` by `shares` and emits an `OperatorSharesIncreased` event
-    function _increaseOperatorShares(address operator, address staker, IStrategy strategy, uint256 shares) internal {
-        operatorShares[operator][strategy] += shares;
-        emit OperatorSharesIncreased(operator, staker, strategy, shares);
+    // @notice Increases `operator`s delegated nonNormalizedShares in `strategy` by `nonNormalizedShares` and emits an `OperatorSharesIncreased` event
+    function _increaseOperatorShares(address operator, address staker, IStrategy strategy, uint256 nonNormalizedShares) internal {
+        nonNormalizedOperatorShares[operator][strategy] += nonNormalizedShares;
+        emit OperatorSharesIncreased(operator, staker, strategy, nonNormalizedShares);
     }
 
-    // @notice Decreases `operator`s delegated shares in `strategy` by `shares` and emits an `OperatorSharesDecreased` event
-    function _decreaseOperatorShares(address operator, address staker, IStrategy strategy, uint256 shares) internal {
+    // @notice Decreases `operator`s delegated nonNormalizedShares in `strategy` by `nonNormalizedShares` and emits an `OperatorSharesDecreased` event
+    function _decreaseOperatorShares(address operator, address staker, IStrategy strategy, uint256 nonNormalizedShares) internal {
         // This will revert on underflow, so no check needed
-        operatorShares[operator][strategy] -= shares;
-        emit OperatorSharesDecreased(operator, staker, strategy, shares);
+        nonNormalizedOperatorShares[operator][strategy] -= nonNormalizedShares;
+        emit OperatorSharesDecreased(operator, staker, strategy, nonNormalizedShares);
     }
 
     /**
-     * @notice Removes `shares` in `strategies` from `staker` who is currently delegated to `operator` and queues a withdrawal to the `withdrawer`.
-     * @dev If the `operator` is indeed an operator, then the operator's delegated shares in the `strategies` are also decreased appropriately.
+     * @notice Removes `nonNormalizedShares` in `strategies` from `staker` who is currently delegated to `operator` and queues a withdrawal to the `withdrawer`.
+     * @dev If the `operator` is indeed an operator, then the operator's delegated nonNormalizedShares in the `strategies` are also decreased appropriately.
      * @dev If `withdrawer` is not the same address as `staker`, then thirdPartyTransfersForbidden[strategy] must be set to false in the StrategyManager.
      */
     function _removeSharesAndQueueWithdrawal(
         address staker, 
         address operator,
         address withdrawer,
         IStrategy[] memory strategies, 
-        uint256[] memory shares
+        uint256[] memory nonNormalizedShares
```

Comment:
potentially might want to think about how to make UX better here so external entities don't think about normalization.

we could denormalize inside of this function, and return the normalized withdrawn shares during completion for example

Reply:
if you do that then you have to do more lookups when the withdrawal is completed.
Specifically, you would need to look up the conversion ratio both when the withdrawal was queued and at the end of slashability, instead of just the latter.
what you're suggesting sounds more like what I had in the first version https://github.com/Layr-Labs/eigenlayer-contracts/pull/528 if you want to take a look at that.
there could also be a way to format things to avoid this? or the 2-lookup tradeoff could be worth it.